packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -759,10 +759,10 @@ contract DeployImplementations is Script {
     // | AnchorStateRegistry     | Yes     | Bespoke                           | No         |
     // | FaultDisputeGame        | No      | Bespoke                           | No         | Not yet supported by OPCM
     // | PermissionedDisputeGame | No      | Bespoke                           | No         |
-    // | DelayedWETH             | Yes     | Two bespoke (one per DisputeGame) | No         |
+    // | DelayedWETH             | Yes     | Two bespoke (one per DisputeGame) | Yes        |
```

Comment:
This one is a bit nuanced, but I think "yes" is still ok: the bond withdrawal delay is immutable, so this implementation can be (and is) reused by OPCM since we only support deploy standard chains. But someone that wants a different delay needs to deploy a new implementation. Perhaps we add a note somewhere in this comment block to clarify that this is "MCP Ready for standard chains"?

Reply:
[ce93488](https://github.com/ethereum-optimism/optimism/pull/12128/commits/ce93488b223799474d53bb1535c6387982f3519d)